interface/backend/cases/tests.py

```diff
@@ -118,3 +118,21 @@ def test_update_nodule_lung_orientation(self):
         self.client.post(url, json.dumps({'lung_orientation': 'NONE'}), 'application/json')
         nodule.refresh_from_db()
         self.assertEquals(nodule.lung_orientation, enums.LungOrientation.NONE.value)
+
+    def test_candidates_mark(self):
+        candidate = CandidateFactory()
+        url = reverse('review-candidate', kwargs={'candidate_id': candidate.id})
+        self.client.post(url, json.dumps({'review_result': enums.CandidateReviewResult.MARKED}), 'application/json')
+
+        candidate.refresh_from_db()
+
+        self.assertEquals(candidate.review_result, enums.CandidateReviewResult.MARKED)
+
+    def test_candidates_dismiss(self):
+        candidate = CandidateFactory()
+        url = reverse('review-candidate', kwargs={'candidate_id': candidate.id})
+        self.client.post(url, json.dumps({'review_result': enums.CandidateReviewResult.DISMISSED}), 'application/json')
+
+        candidate.refresh_from_db()
+
+        self.assertEquals(candidate.review_result, enums.CandidateReviewResult.DISMISSED)
```

Comment:
Ohh, I see the tests were just moved over here :) Is this a better location for them?

Reply:
I just saw `test_update_nodule_lung_orientation` located here and used it as an example. But, ideally, we would test both aspects - the server response and result state in DB. The thing is - I am not sure what is the test coverage policy in our project at this stage.

Reply:
Yes, I do now see the tests are a bit different. What issues did you run into that you decided to remove the above ones?

Reply:
The API has been changed. It used to return only a message about marking\dismissing, I have changed it to return the updated candidate. It's not like I was not able to fix the test because of some issue. Just found an example with `test_update_nodule_lung_orientation` and added the same type of test instead of old one.